cilib/enums.py

```diff
@@ -17,7 +17,7 @@
 # we're working on the 1.24 GA. Set this value to '1.25' sometime between the
 # final RC and GA so we don't get pre-release builds (e.g. 1.24.1-alpha.0) in
 # our 1.24 tracks.
-K8S_NEXT_VERSION = "1.25"
+K8S_NEXT_VERSION = "1.26"
```

Comment:
Some previous adjustments to this line
* https://github.com/charmed-kubernetes/jenkins/pull/892
* https://github.com/charmed-kubernetes/jenkins/pull/764
* https://github.com/charmed-kubernetes/jenkins/commit/ff5f59c10f9a63fef65783452ec96a4650e307dd

Reply:
maybe @kwmonroe can help us know where in the docs adjustment PR https://github.com/charmed-kubernetes/jenkins/pull/975 we should change these values since he seems to do it most often